apps/microtvm/arduino/template_project/tests/test_arduino_microtvm_api_server.py

```diff
@@ -105,9 +106,8 @@ def test_auto_detect_port(self, sub_mock):
         assert handler._auto_detect_port(self.DEFAULT_OPTIONS) == "/dev/ttyACM0"
 
         # Test it raises an exception when no board is connected
-        sub_mock.return_value.stdout = bytes(self.BOARD_DISCONNECTED_V18, "utf-8")
-        with pytest.raises(microtvm_api_server.BoardAutodetectFailed):
-            handler._auto_detect_port(self.DEFAULT_OPTIONS)
+        sub_mock.return_value.stdout = bytes(self.BOARD_CONNECTED_V21, "utf-8")
```

Comment:
I think now comment "# Test it raises an exception when no board is connected" in line 108 must go to above line 112?

Reply:
Oops, fixed the comments